app/views/meal_plans/show.html.erb

```diff
@@ -48,14 +48,13 @@
       <li>
         <%= detail_display(detail) %>
         (<%= link_to detail.recipe.title, detail.recipe %>)
-        <%= link_to '',
+        <%= link_to MaterialIcon.new(icon: :plus_square, title: 'Add to shopping list', size: :medium).render,
                     shopping_list_item_builders_path(
                       shopping_list_id: current_user.shopping_lists.default&.id,
-                      ingredient_ids: [detail.id]
+                      ingredient_ids: detail.id
```

Comment:
Fixes bug where this failed parsing in the controller. I didn't even know this was a bug for a loooooong time. That goes to show how rarely i used this feature.